.github/workflows/release.yml

```diff
@@ -157,4 +157,5 @@ jobs:
           MATURIN_PYPI_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
         with:
           command: upload
+          maturin-version: v1.7.1
```

Comment:
This seems fine, but perhaps we should upgrade to [1.7.3, which is supposed to fix this bug.](https://github.com/PyO3/maturin/pull/2229)

Reply:
Oh they fixed that quick! Great I will try without this change and see if it fixes it.